pfcpiface/upf.go

```diff
@@ -116,6 +122,11 @@ func NewUPF(conf *Conf, fp fastPath) *upf {
 		fastPath:         fp,
 		dnn:              conf.CPIface.Dnn,
 		reportNotifyChan: make(chan uint64, 1024),
+		recoveryTime:     time.Now(),
+		enableHBTimer:    conf.EnableHBTimer,
+		hbMaxRetries:     conf.HbMaxRetries,
+		hbInterval:       time.Duration(conf.HeartBeatInterval) * time.Millisecond,
+		hbRespDuration:   time.Duration(conf.HeartBeatRespDuration) * time.Millisecond,
```

Comment:
Have default values in code if not provided. When just bool to enable is provided

Reply:
Done.